src/layouts/ReviewRequest/components/ManageReviewerModal/ManageReviewerModal.tsx

```diff
@@ -0,0 +1,111 @@
+} from "@chakra-ui/react"
+import { Button, ModalCloseButton } from "@opengovsg/design-system-react"
+import _ from "lodash"
+import Select, { PropsValue, useStateManager } from "react-select"
+
+import { User } from "types/reviewRequest"
+
+export interface ManageReviewerModalProps extends ModalProps {
+  selectedAdmins: User[]
+  admins: User[]
+}
+
+// NOTE: This is with reference to
+// https://github.com/JedWatson/react-select/blob/master/packages/react-select/src/types.ts
+// The initial type is PropsValue, which is either a single value or a multi-value.
+// We need to narrow the type ourself to prove to the TS compiler that this is a list.
+const isList = (admins: PropsValue<unknown>): admins is unknown[] => {
+  // NOTE: Required as SingleValue can be null
+  if (!admins) {
+    return false
+  }
+
+  return !!(admins as User[]).length
+}
+
+export const ManageReviewerModal = ({
+  selectedAdmins,
+  admins,
+  ...props
+}: ManageReviewerModalProps): JSX.Element => {
+  const { onClose } = props
+  const { value: updatedSelectedAdmins, ...rest } = useStateManager({
+    defaultValue: selectedAdmins,
+    options: admins,
+    isMulti: true,
+    isClearable: false,
+  })
+
+  const selectedAdminsList = isList(updatedSelectedAdmins)
+    ? updatedSelectedAdmins
+    : []
+  const remainingAdmins = _.difference(admins, selectedAdminsList)
+
+  return (
+    <Modal {...props}>
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader>
+          <Text textStyle="h4" color="text.title.alt">
+            Manage Reviewers
+          </Text>
+        </ModalHeader>
+        <ModalCloseButton />
+
+        <ModalBody>
+          <VStack spacing="0.75rem" align="flex-start">
+            <Text textStyle="subhead-1">
+              Add or remove Admins who can approve this request
+            </Text>
+            <Select
+              value={updatedSelectedAdmins}
+              {...rest}
+              components={{
+                Input: () =>
+                  remainingAdmins.length > 0 ? (
+                    <Text
+                      ml="0.25rem"
+                      textDecoration="underline"
+                      color="text.link.default"
+                      textStyle="body-2"
+                    >{`+${remainingAdmins.length} more`}</Text>
+                  ) : null,
+              }}
+              styles={{
+                // NOTE: Setting minimum height so that it is same size as button
+                container: (base) => ({
+                  ...base,
+                  width: "100%",
+                  minHeight: "2.75rem",
+                }),
+                control: (base) => ({ ...base, height: "100%" }),
+                // NOTE: Don't allow removal if there is only 1 selected admin
+                multiValueRemove: (base) => {
+                  const canRemove = selectedAdminsList.length > 1
+                  return { ...base, display: canRemove ? "inherit" : "none" }
+                },
```

Comment:
Just curious, would we get to a state where the error toast is triggered like how it is shown on Figma?

Reply:
could you point me to the error toast section?

Reply:
but initial thoughts are that it shouldn't be possible, at least from the UI, to remove the last admin.

Reply:
This is the Figma link: https://www.figma.com/file/7OtNYkc4s9BkFk05ConEbd/%E2%AD%90-V1.5?node-id=2795%3A80957

Screenshot:
<img width="950" alt="Screenshot 2022-10-04 at 09 36 02" src="https://user-images.githubusercontent.com/27919917/193715786-7e8305d2-e8e2-45c5-99a6-e6b0271bbd51.png">

Reply:
When we try to uncheck last admin but that's not possible under the new ui (no X, drop downs don't have checkbox) so idt this shows heh

Reply:
If @NatMaeTan is ok then I think this is ok hahaa

Reply:
as discussed during sync - got impl error!!! should show **all** admins; will fix

Reply:
tracked by #1108. will raise a separate PR for this, won't fix here.